### PR TITLE
Update the widget to distinguish between fees with/without VAT

### PR DESCRIPTION
**Pull request by filip-figiel-alma**
[ClickUp ticket](https://app.clickup.com/t/20427503/SCA-958)

### general
- new flags: `merchant_settings.customer_vat_rate` and `merchant_settings.merchant_vat_rate`
- new L10n messages
### left-hand side
- display fixed and variable fees with and without VAT
- limit the customer fee input to respect maximum interest rates after including VAT
- display customer fee with tax below the input
### right-hand side:
- display fees with and without VAT
- adjust the position of the maximum fee label so that it doesn't overlap with other labels
- put maximum fee label above other labels, so that the on-hover tooltip is accessible.
- calculate the fee sharing bar using the fees with VAT
- indicate when customer fee reached the maximum value
- fix the alignment of the "?" icon
- provide a new example
- use the Parser library to parse and display the example text with more flexibility

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -247,7 +247,11 @@ <h4>Pays Acceptés</h4>
                     "merchant_fee_variable": 100,
                     "min_purchase_amount": 300
                 }
-            ]
+            ],
+            "merchant_settings": {
+              "customer_vat_rate": 0.2000,
```
Comment (MarcAntoineGautier) [inline public/index.html:252]: Are we sure about these values?

**Review by MarcAntoineGautier — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (filip-figiel-alma) [reply public/index.html:252]: Yup, it's just sample data. This won't be present in the widget code

File: src/Views/FeePlan.elm [hunk 2]
```diff
@@ -357,109 +460,174 @@ showInterestPanel l10n { customer_fee_variable, merchant_fee_variable, maximum_i
...
+        blueParser =
+            Parser.succeed ExampleTextBlue
+                |. Parser.symbol "<blue>"
+                |= (Parser.chompUntil "</blue>" |> Parser.getChompedString)
+                |. Parser.symbol "</blue>"
+
+        fallbackTagParser =
+            Parser.succeed (\a b -> ExampleText (a ++ b))
+                |= (Parser.symbol "<" |> Parser.getChompedString)
+                |= (Parser.chompUntilEndOr "<" |> Parser.getChompedString)
```
Comment (filip-figiel-alma) [inline src/Views/FeePlan.elm:610]: wondering if we can avoid having to do `a ++ b` here and in the `textParser`. But it works well as it is

**Comment by filip-figiel-alma** [comment]
@PascalLeMerrer all done

**Pull request merged**